test/controllers/update/overseas.entity.presenter.controller.spec.ts

```diff
@@ -90,6 +92,13 @@ describe("OVERSEAS ENTITY PRESENTER controller", () => {
       expect(resp.status).toEqual(500);
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test(`renders the ${UPDATE_PRESENTER_PAGE} page for remove`, async () => {
+      mockGetApplicationData.mockReturnValueOnce({ });
+      const resp = await request(app).get(`${OVERSEAS_ENTITY_PRESENTER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`);
+
+      expect(resp.status).toEqual(302);
```

Comment:
This should be a 200 rather than a 302 (see original test on line 79). It looks like the middleware function `navigation.hasOverseasEntityNumber` is sending a redirect before it hits the controller so the code this is supposed to test is never being called.
This is defined in route: 
```
router.route(config.OVERSEAS_ENTITY_PRESENTER_URL)
  .all(
    authentication,
    companyAuthentication,
    navigation.hasOverseasEntityNumber
  )
  .get(overseasEntityPresenter.get)
  .post(...validator.presenter, checkValidations, overseasEntityPresenter.post);
```

Reply:
200